doc/classes/OS.xml

```diff
@@ -682,15 +690,19 @@
 			The current screen index (starting from 0).
 		</member>
 		<member name="exit_code" type="int" setter="set_exit_code" getter="get_exit_code">
+			The exit code the engine returns to the OS upon exiting the main loop.
 		</member>
 		<member name="keep_screen_on" type="bool" setter="set_keep_screen_on" getter="is_keep_screen_on">
+			If [code]true[/code], the engine tries to keep the screen on while the game is running. Useful on mobile.
 		</member>
 		<member name="low_processor_usage_mode" type="bool" setter="set_low_processor_usage_mode" getter="is_in_low_processor_usage_mode">
+			If [code]true[/code], the engine tries to optimize for low processor usage by only refreshing the screen if needed instead of always. Useful to improve battery consumption on mobile, e.g. laptops or smartphones/tablets.
```

Comment:
Drop "instead of always" - it is redundant with "if needed".
I think you can drop the "e.g." part and just say "on mobile platforms". People know what those are.